arbor/morph/embed_pwlin.cpp

```diff
@@ -93,9 +93,9 @@ mcable_list data_cmp(const branch_pw_ratpoly<1, 0>& f, unsigned bid, double val,
     mcable_list L;
     const auto& pw = f.at(bid);
     for (const auto& piece: pw) {
-        auto extents = piece.first;
-        auto left_val = piece.second(0);
-        auto right_val = piece.second(1);
+        auto extents = piece.interval;
+        auto left_val = piece.element(0);
+        auto right_val = piece.element(1);
```

Comment:
Here the call operator is used....

Reply:
Here it is evaluating the polynomial at the ends…

Reply:
(But, this could also in principle use the stored interpolant values directly.)

Reply:
Ok